content/community/governance/gcb/index.md

```diff
@@ -0,0 +1,102 @@
+
+Efficient communication between the GCB (Community Board) and both the GEB (Executive Board) and the GTB (Technical Board) requires at least one representative from GEB and from GTB to be also members of the GCB. It is recommended that at least one member of each of the other Boards are also members of the GCB, at all times.
+
+In summary, the goals of the GCB are:
+
+* discuss and summarize scientist (user) feedback to help guide the technical development of the Galaxy platform;
+* communicate scientist (user) vision and feedback to the GTB and GEB;
+* develop proposals to advance scientist (user) goals in the Galaxy community.
+
+## Community Special Interest Groups (SIGs)
+
+The GCB is a governance structure representing groups that are not primarily technical in nature, such as existing Proteomics and Microbial communities, though engaging in capacity building, toolkit development, outreach and training. The term *[Special Interest Group](/community/sig)* (SIG) is proposed as a term referring to these kinds of groups. No specific restrictions are proposed, other than making a useful contribution to the Project, so collaborating commercial users could form and manage their own SIG and gain access to the Roadmap process. 
+
+Active contributing communities are encouraged to form and govern their own SIG, with a focus on specific shared interests. Each SIG may elect one member to represent their interests on the GCB. This is analogous to the Technical Board, consisting of representatives of the Working Groups (WG).
+
+### Characteristics of SIGs
+
+* Primarily focused on community development: as distinct from the more technical issues managed by WGs.
+* Flexible model: with simple criteria such as evidence of genuine community support, hub updates and self-regulation,
+for ongoing recognition, easy to create and to dissolve, can be transient
+* Some supporting administrative infrastructure will be provided by the Galaxy Project 
+  * mailing list
+  * website infrastructure and a place on the Galaxy hub to present the Community
+  * presentation slot at the annual GCC conference
+  * Chat channel/room
+* Govern their own activities such as organising meetings, developing training material and tools, maintaining the Hub pages, and choosing a representative for the GCB.
+* Internally organise themselves to gather and articulate their own needs and requirements as part of the Galaxy Project roadmap and (competitive grant) processes.
+
+### Requirements for SIGs
+
+* A SIG must have 2 or more members to be viable.
+* SIGs must maintain a page on the Hub with at least:
```

Comment:
```suggestion
* Must maintain a page on the Hub with at least:
```

Reply:
add link to SIGs

Reply:
minor thing but I'd put the hub page req section underneath the single-line requirements